scripts/prepare.js

```diff
@@ -14,28 +14,16 @@ if (process.env.npm_config_zmq_external == "true") {
 
 function buildZMQ(scriptPath, zmqDir) {
   console.log("Building libzmq for " + process.platform);
-  try {
-    const child = spawnSync(scriptPath, [ZMQ, ARCH], {
-      cwd: process.cwd(),
-      env: process.env,
-      stdio: ['inherit', 'inherit', 'pipe'],
-      encoding: 'utf-8',
-    });
-    if (child.stderr) {
-      return console.log(Error(child.stderr))
-    }
-    if (child.status !== 0) {
-      return console.error("child process exited with code " + child.status);
-    }
-  } catch(e) {
-    return console.error(e)
+  const child = spawnSync(scriptPath, [ZMQ, ARCH], {
```

Comment:
If you have verified that `shell : true` is needed, then we should quote the path.
```suggestion
  const child = spawnSync(`"${scriptPath}"`, [ZMQ, ARCH], {
```

Reply:
OK, I'll fix that and another issue that I've noticed (Atom removes the execution permit from all the files in `scripts`).

Reply:
`apm` shouldn't change anything related to the scripts (I haven't seen such a thing before). The installation happens by `npm` underneath. The issue may be that I edited the files on Windows. If the permissions are incorrect, please edit and push them to Git.

Reply:
For those of you unbelievers :stuck_out_tongue_closed_eyes: , here's what a see:

```
$ cd
$ rm -rf .atom/packages/Hydrogen/
$ npm_config_zmq_external=true apm install hydrogen
Installing hydrogen to /home/nriesco/.atom/packages ✓
$ ls -la .atom/packages/Hydrogen/node_modules/\@aminya/zeromq/scripts/
total 28
drwxr-xr-x 2 nriesco nriesco 4096 Apr 18 13:51 .
drwxr-xr-x 8 nriesco nriesco 4096 Apr 18 13:51 ..
-rw-r--r-- 1 nriesco nriesco  945 Apr 18 13:51 build_libzmq.sh
-rw-r--r-- 1 nriesco nriesco  659 Apr 18 13:51 cross_compile.sh
-rw-r--r-- 1 nriesco nriesco 1172 Apr 18 13:51 download.js
-rw-r--r-- 1 nriesco nriesco 2890 Apr 18 13:51 prepare.js
-rw-r--r-- 1 nriesco nriesco  616 Apr 18 13:51 publish_docs.sh
```